questions/functionconditions.md

```diff
@@ -22,17 +26,11 @@ First, set up a dummy scoreboard objective:
 
 We're going to use the executing entity to store the result. This can of course be substituted for a fake player or a different entity as well.
 
-Make sure the score is initiated (not null). Easy way to do this is by adding 0:
-
-    scoreboard players add @s success_score 0
-
-----
-
 Then, whenever you want to perform a conditional command:
 
-    execute store success @s success_score if block 73 10 31 stone
-    execute if score @s success_score=1 run say Stone found!
-    execute if score @s success_score=0 run say Stone not found!
+    execute store success #success success_score if block 73 10 31 stone
```

Comment:
in the text above it says

> We're going to use the executing entity to store the result

so we should still do that in the example.

Alternatively we should change the above text to mention that using a fake player is a good idea in this case (and link to the info about the fake player), because it's about checking a block, independently of an entity.

Reply:
also, wtf happened with that syntax? who wrote this completely wrong thing? xD

(tbh it was probably me when I was super tired at some point, wow)

Reply:
> tbh it was probably me when I was super tired at some point, wow)

Lol